resources/lang/ca/activity.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    'login'                         => 'Connectat',
+    'registration'                  => 'Registre',
+    'view_home'                     => 'Visualització de casa',
+    'view_searches'                 => 'Visualització de cerques',
+    'view_lost_people_detail'       => 'Visualització de detalls de persones perdudes',
+    'view_lost_people_detail'       => 'Visualització de detalls de l’usuari',
```

Comment:
Duplicate 'view_lost_people_detail' ???

In case is 'view_user_details' it should be:
'view_user_details' => "Veure detalls de l'usuari"